### PR TITLE
POL-576 Add Azure Savings Realized

### PR DESCRIPTION
**Pull request by nia-vf1**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/savings_realized/azure_savings_realized.pt [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+    summary_template "Savings Realized"
+    detail_template <<-EOS
+    # Savings Realized Per Instance Type Report
+    ![Savings Realized Per Instance Type Chart](https://image-charts.com/chart?{{ with index data 0 }}{{ .chart_dimensions.chart_data }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_size }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_type }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_image }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_title }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_label_position }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_y_axis }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_axis_label }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_line_style }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_line_color }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_data_scale }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_legend_size }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_legend }}{{ end }}&{{ with index data 0 }}{{ .chart_dimensions.chart_axis_format }}{{ end }})
```
[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/azure/savings_realized/azure_savings_realized.pt:443]: I wasn't sure whether we were still only using that new endpoint for testing?

File: cost/azure/savings_realized/azure_savings_realized.pt [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+      field "month" do
+        label "Month"
+      end
+      field "dimension" do
+        label "Amount Type"
+      end
+      field "cost" do
+        label "Cost Amount"
+      end
+      #field "region" do
```
[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/azure/savings_realized/azure_savings_realized.pt:459]: Will remove this

[the remark this answers is not in the document]
Comment (nia-vf1) [reply cost/azure/savings_realized/azure_savings_realized.pt]: Removed

**Review by beagledeltasix — DISMISSED** [review]
Approved

**Review by beagledeltasix — APPROVED** [review]
Approved the new changes

**Pull request merged**